crates/biome_js_analyze/src/lint/nursery/use_default_switch_clause.rs

```diff
@@ -0,0 +1,93 @@
+use std::ops::Not;
+
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_js_syntax::JsSwitchStatement;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Require default cases in switch statements.
+    ///
+    /// Some code conventions require that all switch statements have a default case. The thinking is that it’s better
+    /// to always explicitly state what the default behavior should be so that it’s clear whether or not the developer
+    /// forgot to include the default behavior by mistake.
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```js,expect_diagnostic
+    /// switch (a) {
+    ///     case 1:
+    ///         /* code */
+    ///         break;
+    /// }
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```js
+    /// switch (a) {
+    ///     case 1:
+    ///         /* code */
+    ///         break;
+    ///
+    ///     default:
+    ///         /* code */
+    ///         break;
+    /// }
+    /// ```
+    ///
+    /// ```js
+    /// switch (a) {
+    /// }
```

Comment:
Just wondering: why this one is valid?

Reply:
when I first wrote the rule, the aim was to cover 100% of [ESLint's default-case rule tests](https://github.com/eslint/eslint/blob/main/tests/lib/rules/default-case.js#L35), but since I removed the `commentPattern` option that doesn't matter anymore, so now we can just treat it as an invalid case